From 17c3a1a1a22c468f3cfe4801e034fa76dbddb9aa Mon Sep 17 00:00:00 2001
From: Matthew Parsons <mparsons@mindless.com>
Date: Sat, 28 Jul 2018 16:18:47 +0000
Subject: [PATCH] joyent/smartos-live#690 trivial imgadm(1M) manpage fix (gz ->
 gzip) Reviewed by: Joshua M. Clulow <jmc@joyent.com> Approved by: Joshua M.
 Clulow <jmc@joyent.com>

---
 src/img/man/imgadm.1m.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/img/man/imgadm.1m.md b/src/img/man/imgadm.1m.md
index b56d6930..1d6d9e90 100644
--- a/src/img/man/imgadm.1m.md
+++ b/src/img/man/imgadm.1m.md
@@ -361,7 +361,7 @@ UUID.
                            then "PATH.imgmanifest" and "PATH.zfs[.EXT]" are
                            created.
             -c <comp>, --compression=<comp>
-                           One of "none", "gz", "bzip2" or "xz" for the compression
+                           One of "none", "gzip", "bzip2" or "xz" for the compression
                            to use on the image file, if any. Default is "none".
             -i             Build an incremental image (based on the "@final"
                            snapshot of the source image for the VM).
-- 
2.21.0

